soc/intel/intel_socfpga_std/Kconfig

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# Description:
+# Adding support to Cyclone V SoC FPGA
```

Comment:
not really sure why there is what looks to be a changelog here and in the other files?